reactive-review/AppFlow.adoc

```diff
@@ -0,0 +1,12 @@
+:gls_prefix:
+
+// Do not modify section headers, as they are used by flamel and the translation process.
+= pass:a,n[{gls_res_outcomes}]
+
+You should be able to create Quarkus Conference application, by using Reactive Architecture and Principles.
+
+== Proposed App Flow
+
+* Create `session mS` a `Reactive` application using Quarkus.
+* Create an `asynchronous REST` endpoint by using RESTeasy Reactive (Attaching an image for the same).
```

Comment:
Also, If possible we might want to make the `session` app read data from a database. The database access should be reactive too.

Reply:
We want students to use the `@Blocking`/`@NonBlocking` annotations. 

An example of this might be a REST endpoint in the `session` service that returns a Uni stream but performs a blocking operation. The blocking operation might be a "sleep" or some calculation that keeps the event-loop thread blocked for a couple of seconds. To avoid blocking the event loop, students will have to use the `@Blocking` annotation